docs/UserGuide.adoc

```diff
@@ -76,53 +75,105 @@ image::FontSizeUI.png[width="540"]
 * *Reset Size:* kbd:[CTRL] + kbd:[R] (kbd:[CMD] + kbd:[R] on MacOS)
 ****
 
-Keyboard shortcuts have been provided for your convenience.
+****
+**Option 2: Change Font Size Buttons (Top Right Hand Corder)**
 
-[NOTE]
-====
-There is a limit to how much your font sizes can increase or decrease. Also, font sizes will increase for the
-contacts entries and the todo list's entries.
-====
+image::FontSizePlusMinusButtons.png[width="540"]
+
+* Increase Size: kbd:[+]
+* Decrease Size: kbd:[-]
+* Reset Size: kbd:[R]
+****
+
+****
+**Option 3: Handtype command into CLI**:
+
+image::FontSizeCLI.png[width="540"]
+
+* Increase Size Command: `fontsize increase`
+* Decrease Size Command: `fontsize decrease`
+* Reset Size Command: `fontsize reset`
+****
 
 === Viewing help : `help`
+If you have trouble figuring out how to do something, or would like to find out about more features, you can click
+on the **help** menu button
+
```

Comment:
I think you can mention that the command can be used here? Don't seem to see it here

Reply:
Yup thanks for pointing that out